client/src/reducers/frames.js

```diff
@@ -35,7 +40,7 @@ export default (state = defaultState, action) =>
 
             case PATCH_FRAME:
                 Object.assign(
-                    draft.items.find(frame => frame.id === action.id),
+                    draft.items.find(frame => frame.id === action.id) || {},
```

Comment:
Why is this needed? Seems like this fallback is hiding potential problems with patches to a non-existing frame

Reply:
The reason I put it in was because selecting an old frame item in the history triggers that query. So if you click the play button before it's done loading, patch fails because we already removed the frame. I'd have to investigate more about what causes the query to trigger. Maybe instead of executing it should just set the query in the editor (that'd be my preference and expectation).

In any case, I'm happy to remove this if you think it has the potential to lead to more problems?

Reply:
It's probably an unrelated issue, but happy to merge this as is. We can worry about potential issues later, esp. since I've dragged this PR for so long.

Thank you for your contribution and patience